### DISCUSSION TITLE
Issue when using Behringer U-Phoria1820

### DISCUSSION THREAD

**Discussion by baaccs** [apohl79/audiogridder#1110] [Help]
I am trying Audiogridder for the first time instead of Sound grid because I love the idea of it working with most plugins. I was having an issue of sorts and for some reason it doesn't like something with the Behringer U-Phoria 1820. The interface works flawlessly without Audio Gridder but after about 3 to 5 plugins it would start overloading the DAW computer but the server was fine. I tried with just the internal windows audio output, and it works great. WEIRD!!! Just using a a 2gig 2 Core i3 that's an older computer and it took a project I was at 90% DSP to 18% DSP WOW, loving it, hopefully can find out what is causing the issue when using the interface. Also I am using Samplitude ProX 7

**Comment by apohl79** [comment]
Try to increase the sample buffer in the DAW. You might also have to use buffering in the AG plugin as well.
